rllib/policy/torch_policy_v2.py

```diff
@@ -878,8 +882,28 @@ def get_initial_state(self) -> List[TensorType]:
 
     @override(Policy)
     @DeveloperAPI
-    def get_state(self) -> Union[Dict[str, TensorType], List[TensorType]]:
+    @OverrideToImplementCustomLogic_CallToSuperRecommended
+    def get_state(self) -> PolicyState:
+        # Legacy Policy state (w/o torch.nn.Module and w/o PolicySpec).
         state = super().get_state()
+
+        # Add this Policy's spec so it can be retreived w/o access to the original
+        # code.
+        state["policy_spec"] = PolicySpec(
```

Comment:
Add PolicySpec AND - if possible - native keras/torch saved model files to returned state.